common/types/chain.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+import "fmt"
+
+// TODO : Remove ID from txmgr/types to move to common/types
```

Comment:
Ideally I would also like to merge this with https://github.com/smartcontractkit/chainlink/blob/d5b44c27abd138ee6be4be02f2bc243a52f03e1a/core/chains/constraints.go#L4.

Perhaps @jmank88 can weigh in.

Reply:
sure if they are similar enough